### PR TITLE
Adding Bubble_Sort

### LINKED ISSUE #1170: Bubble sort in kotlin
Sorry, But can you please Re-assign it to me. Last time i Don't  know how to use git and this messed up things in first submission. But now I learned a lot and i want to submit it again.

### PR DESCRIPTION
**Pull request by shubhammantri1**
Fixes #1170  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Adding Bubble_Sort in Kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Bubble_Sort.kt

### CODE REVIEW

File: Bubble_Sort/Bubble_Sort.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+fun bubbleSort(list: MutableList<Int>):List<Int> {
+    // flag keeps track if any swap happened or not, if swap not happened then array is sorted and it will break out of the loop
+    // and will save time from running for loop unnecessarily
+    var flag: Int = 0
+    for(i in 0 until (list.size - 1)) {
+        for (j in 0 until (list.size - i - 1)) {
+            if(list[j] > list[j + 1]) {
+                var tmp = list[j + 1]
+                list[j + 1] = list[j]
+                list[j]  = tmp
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.kt]: There are 2 spaces before = in this line.
Please remove one space.

File: Bubble_Sort/Bubble_Sort.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+//Driver Function for Bubble Sort
+fun main(args:Array<String>) {
+    val numbers = mutableListOf(9, 5, 25, 9, 2, 1)
+    println("Unsorted: $numbers")
+    val sortedlist = bubbleSort(numbers)
+    println("Sorted: $sortedlist")
+}
+
+// Input: Unsorted: [90, 5, 25, 19, 2, 1]
+//Output: Sorted: [1, 2, 5, 19, 25, 90]
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.kt]: Add a space after //

File: Bubble_Sort/Bubble_Sort.kt [hunk 3]
```diff
@@ -0,0 +1,30 @@
+//Function for Bubble Sort
+fun bubbleSort(list: MutableList<Int>):List<Int> {
+    // flag keeps track if any swap happened or not, if swap not happened then array is sorted and it will break out of the loop
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.kt]: Please add more words of this line to the second line of comment.

File: Bubble_Sort/Bubble_Sort.kt (hunk 1 above)
Comment (shubhammantri1) [reply Bubble_Sort/Bubble_Sort.kt]: Done

File: Bubble_Sort/Bubble_Sort.kt (hunk 2 above)
Comment (shubhammantri1) [reply Bubble_Sort/Bubble_Sort.kt]: Done

File: Bubble_Sort/Bubble_Sort.kt (hunk 3 above)
Comment (shubhammantri1) [reply Bubble_Sort/Bubble_Sort.kt]: Done

File: Bubble_Sort/Bubble_Sort.kt [hunk 4]
```diff
@@ -0,0 +1,32 @@
+//Function for Bubble Sort
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.kt]: Space after // here also

File: Bubble_Sort/Bubble_Sort.kt [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+                flag++
+            }
+        }
+        if(flag == 0)
+            break
+    }
+    return list
+}
+
+//Driver Function for Bubble Sort
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.kt]: Space after // here also

File: Bubble_Sort/Bubble_Sort.kt [hunk 6]
```diff
@@ -0,0 +1,32 @@
+//Function for Bubble Sort
+fun bubbleSort(list: MutableList<Int>):List<Int> {
+    /* flag keeps track if any swap happened or not,
+     if swap not happened then array is sorted and it will break
+      out of the loop and will save time from
+      running for loop unnecessarily.*/
+    var flag: Int = 0
+    for(i in 0 until (list.size - 1)) {
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.kt]: space after for

File: Bubble_Sort/Bubble_Sort.kt [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        for (j in 0 until (list.size - i - 1)) {
+            if(list[j] > list[j + 1]) {
```
Comment (MastersAbh) [inline Bubble_Sort/Bubble_Sort.kt]: space after if

Comment (shubhammantri1) [reply Bubble_Sort/Bubble_Sort.kt]: Done

File: Bubble_Sort/Bubble_Sort.kt (hunk 6 above)
Comment (shubhammantri1) [reply Bubble_Sort/Bubble_Sort.kt]: Done

File: Bubble_Sort/Bubble_Sort.kt (hunk 5 above)
Comment (shubhammantri1) [reply Bubble_Sort/Bubble_Sort.kt]: Done

File: Bubble_Sort/Bubble_Sort.kt (hunk 4 above)
Comment (shubhammantri1) [reply Bubble_Sort/Bubble_Sort.kt]: Done

File: Bubble_Sort/Bubble_Sort.kt (hunk 7 above)
Comment (amaaniqbal) [reply Bubble_Sort/Bubble_Sort.kt]: You added space after the second if. Please add space before first one too.

Comment (shubhammantri1) [reply Bubble_Sort/Bubble_Sort.kt]: Done

**Pull request merged**